streams/src/main/java/org/apache/kafka/streams/processor/StateRestoreListener.java

```diff
@@ -37,6 +40,11 @@
  * These two interfaces serve different restoration purposes and users should not try to implement both of them in a single
  * class during state store registration.
  *
+ * <p>
+ * Also note that standby tasks restoration process are not monitored via this interface, since a standby task keep
```

Comment:
```suggestion
 * Also note that the restoration process of standby tasks is not monitored via this interface, since a standby task keeps
```

Reply:
ack

Reply:
`restoration` -> `update` ?

Reply:
> since a standby task keeps getting data from the changelogs written by the active stream tasks and hence would not complete restoration at all.

-> `since a standby task does not actually _restore_ state, and updating a standby does never finish.